utils/helpers.ts

```diff
@@ -17,6 +17,10 @@ import {
   Metadata,
 } from './types';
 
+export const removeEndingBackslash = (uri: string): string => {
```

Comment:
This is required only because of the function below I think? So we may be able to remove this method if we remove the ending backslash in the `uriToHttp` method? @ECWireless

Reply:
Removed in 7ad31712814cdd834f524e51ec4d990ca0400d4d